template/interface_cpp.njk

```diff
@@ -19,28 +19,30 @@
 #include "core/js_type_traits.h"
 #include "core/native_type_traits.h"
 
-void {{name}}Bridge::Init(Napi::Env env, Napi::Object exports) {
+// FIXME: should be distinguished whether it is interface or not.
+{% for definition in definitions %}
+void {{definition.name}}Bridge::Init(Napi::Env env, Napi::Object exports) {
```

Comment:
I think these definitions should be defined in each different files.

Reply:
i will change it.